src/lib/ydocker/main_dialog.rb

```diff
@@ -148,22 +156,33 @@ def selected_container
       Docker::Container.get(selected)
     end
 
-    def stop_container
-      return unless (Yast::Popup.YesNo(_("Do you really want to stop the running container?")))
-      selected_container.stop!
-      return unless (Yast::Popup.YesNo(_("Do you want to remove the container?")))
-      selected_container.delete
+    def start_container
+      return unless (Yast::Popup.YesNo(_("Do you really want to start the container?")))
+      selected_container.start!
 
       redraw_containers
     end
 
-    def kill_container
-      return unless (Yast::Popup.YesNo(_("Do you really want to kill the running container?")))
-      selected_container.kill!
-      return unless (Yast::Popup.YesNo(_("Do you want to remove the container?")))
-      selected_container.delete
+    def stop_container
+      if Yast::Popup.YesNo(_("Do you really want to stop the running container?"))
+        selected_container.stop!
+        redraw_containers
+      end
+      if Yast::Popup.YesNo(_("Do you want to remove the container?"))
```

Comment:
does it make sense if answers are "no" and "yes"? it means that you do not stop container but remove it